### ISSUE TITLE
Message with enum field containing keyword generates illegal code.

### ISSUE THREAD

**Issue by peterhuene** [stepancheg/rust-protobuf#295]
## Repro Steps

Compile the following proto file:

```
syntax = "proto3";

package Example;

enum E {
  in = 0; // Uses the Rust 'in' keyword
}

message M {
  E e = 1;
}
```

## Expected Results

The generated Rust code compiles.

## Actual Results

The enum `E` is generated correctly:

```
#[derive(Clone,PartialEq,Eq,Debug,Hash)]
pub enum E {
    value_in = 0,
}
```

However, the following methods of `M` reference `E::in` instead of `E::value_in`:

* `clear_e`
* `compute_size`
* `write_to_with_cached_sizes`

This leads to three compilation errors: `expected identifier, found keyword`.

**Comment by peterhuene** [comment]
Thanks for the quick response, @stepancheg!

**Comment by stepancheg** [comment]
You are welcome! Going to backport it to stable branches now.

**Comment by stepancheg** [comment]
Published version `2.0.1`.
